plugins/woocommerce/includes/class-wc-cart-totals.php

```diff
@@ -758,7 +758,7 @@ protected function calculate_item_subtotals() {
 
 		// Prices are not rounded here because they should already be rounded based on settings in `get_rounded_items_total` and in `round_line_tax` method calls.
 		$this->set_total( 'items_subtotal', $items_subtotal );
-		$this->set_total( 'items_subtotal_tax', array_sum( $merged_subtotal_taxes ), 0 );
+		$this->set_total( 'items_subtotal_tax', array_sum( $merged_subtotal_taxes ));
```

Comment:
Add missing space (lint rules)
```suggestion
		$this->set_total( 'items_subtotal_tax', array_sum( $merged_subtotal_taxes ) );
```